src/core.js

```diff
@@ -1228,6 +1228,10 @@ define([
         }).on('parent-question-type-change', function (e) {
             _this.jstree("set_icon", e.childMug.ufid, e.childMug.getIcon());
         }).on('remove-question', function (e) {
+            var currentMug = _this.getCurrentlySelectedMug();
+            if (e.mug && e.mug.parentMug && e.mug.parentMug === currentMug) {
+                _this.displayMugProperties(currentMug);
+            }
```

Comment:
Was there a particular reason this was put at the beginning of this callback? Seems like `selectSomethingOrHideProperties` will possibly call `displayMugProperties` again. Would be good to not display properties more than once.